### PR TITLE
gdx-pay-android-amazon: Fix project setup

### PR DESCRIPTION
**Pull request by MrStahlfelge**
It was not possible to import gdx-pay-android-amazon with gradle because it was no Android library project.

Hence, the following error was thrown when running a game using PurchaseManagerAndroidAmazon:

java.lang.NoClassDefFoundError: com.badlogic.gdx.pay.android.amazon.PurchaseManagerAndroidAmazon

This commit fixes the project setup.

### CODE REVIEW

**Comment by keesvandieren** [comment]
The Android lint fails on the project. Can you try to fix that?

**Comment by MrStahlfelge** [comment]
I added ignore flags to the manifest. The manifest entry is correct how it is, Lint just don't know about Amazon.

**Comment by MrStahlfelge** [comment]
I added some further commits because entitlements where just not working and install() was called to early. I did not use an own branch for these fixes because they are not possible to use without this branch here.

File: gdx-pay-android-amazon/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# InApp purchasing implementation for Amazon
+
+Handles purchases and restores for non-consumable and consumable products, will not work as intended for subscriptions.
+
+## proguard configuration
```
Comment (keesvandieren) [inline gdx-pay-android-amazon/README.md]: Proguard: uppercase

**Comment by MrStahlfelge** [comment]
Hi, you are right about the warnings. 

My goal was to get the project going in the first place without changing much of the existing codebase. Please review my changes to the lib, I did not introduce these problems. I must point out that the current stable version _is not useable_ with Gradle and _it does not work_ with Entitlements. It would be a big step to merge these fixes before improving the comments.

**Comment by keesvandieren** [comment]
Just merged :-)

**Comment by MrStahlfelge** [comment]
Thank you. I will take a look on your remarks and other warnings in the lib soon!

**Comment by MrStahlfelge** [comment]
Hi again, most things were already done in my own branch. I have added some of your other remarks, too: 
https://github.com/MrStahlfelge/gdx-pay/tree/mrstahlfelge

Problem is again, this branch depends on changes I made for the googlebilling implementation so it can't get pulled before reviewing googlebilling. Sorry. ;-)

**Comment by keesvandieren** [comment]
I just merged the other PR as well. Looking forward to further contributions :-).

Regarding Google Billing client: if you have the time / ability, can you add some unit tests? View Google Play implementation for an idea of how to achieve this. 

Thanks in advance!

**Comment by MrStahlfelge** [comment]
I will take a look!

**Pull request merged**